plaidml2/edsl/edsl_test.py

```diff
@@ -7,6 +7,11 @@
 
 import plaidml2 as plaidml
 from plaidml2.edsl import *
+import plaidml2.settings as plaidml_settings
+import plaidml2.exec as plaidml_exec
+
+device = plaidml_settings.get('PLAIDML_DEVICE')
+target = plaidml_settings.get('PLAIDML_TARGET')
```

Comment:
`PLAIDML_DEVICE` and `PLAIDML_TARGET` need to be configured either in a `~/.plaidml2` file on all build machines, or added to the `plan.yml` somehow. @drgone Can you help get this setup on all the build machines?

Reply:
plan.yml seems like a better idea for build machines that have more than one device.

Reply:
In this context, we only need to configure build machines (not those involved with tests) so there should only ever be one device (and probably LLVM or OpenCL CPU is appropriate).

Reply:
got it,  related: added env vars to ops/buildkite, and modified harness to pick it up #493